frontend/src/index.css

```diff
@@ -1,10 +1,21 @@
+* {
```

Comment:
this css will affect all components, this could create a merge conflict

Reply:
well not a merge conflict, just some css chaos

Reply:
we can see what happens and then debug it

Reply:
Should I wait on this then until our meeting?